### ISSUE TITLE
Support additional meta data in personal identity information

### ISSUE THREAD

**Issue by DarqueWarrior** [Azure/static-web-apps#89]
If I were to compare this to Firebase I can store additional meta data like "Display Name" without having to spin up a data store for it. 

Can I store any meta data like "Display Name" in the static web app personal identity information? Or do I have to store that data myself?

**Comment by miwebst** [comment]
Are you looking for information separate from the userDetails field which will be a username or email? https://docs.microsoft.com/en-us/azure/static-web-apps/user-information

Or are you looking for a way to add arbitrary metadata?

**Comment by DarqueWarrior** [comment]
As a user that logs in with Google I don't want to say "Hello someEmail@gmail.com". I want to say "Hello Donovan". To do that I need to ask for and store a "Display Name". Today I would have to spin up and manage my own data store just for simply meta data on the user.  I am looking for something like [this](https://firebase.google.com/docs/auth/web/manage-users#get_a_users_provider-specific_profile_information) from Firebase from Google.

**Comment by miwebst** [comment]
Makes sense, yes we have some information today in the 'userDetails' field and it is accessible both by the clientside code and function code. For Google, Facebook, and AAD the user details will be the email address of the user. For Twitter and Github it will be their handle. People have used this in the past to make a "Hello <someUser>" experience.

**Comment by Zhiyuan-Amos** [comment]
> are you looking for a way to add arbitrary metadata?

Hello, can I know if arbitrary metadata will eventually be supported, which Azure AD currently [support](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/user-flow-add-custom-attributes) on preview?
